db/migrate/20241017011246_active_record_doctor_incorrect_length_validation.rb

```diff
@@ -0,0 +1,23 @@
+class ActiveRecordDoctorIncorrectLengthValidation < ActiveRecord::Migration[7.2]
+  def up
+    change_column :links, :url, :string, limit: 250
+    change_column :invitation_requests, :memo, :string, limit: 255
+    change_column :invitations, :memo, :string, limit: 255
+    change_column :hat_requests, :comment, :text, limit: 65_535
+    change_column :hats, :short_id, :string, limit: 10
+    change_column :categories, :category, :string, limit: 25
+    change_column :stories, :description, :text, limit: 65_535
+    change_column :messages, :body, :text, limit: 65_535
+  end
+
+  def down
+    change_column :links, :url, :string
+    change_column :invitation_requests, :memo, :string
+    change_column :invitations, :memo, :string
+    change_column :hat_requests, :comment, :text, size: :medium
+    change_column :hats, :short_id, :string
+    change_column :categories, :category, :string
+    change_column :stories, :description, :text
```

Comment:
I'm really puzzled why you removed the `size: :medium` from `text` columns on `invitations`, `messages`, etc. Is there a value to that? I like to have the database backstopping that sort of validation with its explicit size, even if the default for a `text` column is `medium`.

Reply:
Hi, it was a mistake. 😞 
Since it is the `down` method for the migration, it should restore to the previous exact status for a rollback.